src/RestfulClient.cs

```diff
@@ -31,6 +31,11 @@ public class RestfulClient : IRestfulClient
         /// </summary>
         public HttpClient Client { get; set; }
 
+        /// <summary>
+        /// Gets or sets a collection of headers that will be added by default to every outgoing request.
+        /// </summary>
+        protected Dictionary<string, string> DefaultHeaders { get; set; }
```

Comment:
```private set```?

Reply:
Also, do you want to store these separately on the ```RestfulClient``` or modify ```HttpClient```'s ```DefaultRequestHeaders``` property?

Reply:
Nope, you know I don't like private setters. On the other question, I was not aware of that property. However, I think the current implementation is simpler and does everything it should do. I don't really see much value going that way.

Reply:
I think it's simpler to have all of the default headers on a single object, instead of spread across one of ```HttpClient```'s properties and one of ```RestfulClient```'s properties.